src/canister/individual_user_template/src/api/cycle_management/return_cycles_to_user_index_canister.rs

```diff
@@ -1,15 +1,14 @@
-use ic_cdk::api::management_canister::{main, provisional::CanisterIdRecord};
+use ic_cdk::{api::{canister_balance128, management_canister::{main, provisional::CanisterIdRecord}}, caller};
 use ic_cdk_macros::update;
 use shared_utils::{
-    common::types::known_principal::KnownPrincipalType,
+    common::{types::known_principal::KnownPrincipalType, utils::permissions::is_caller_controller},
     constant::INDIVIDUAL_USER_CANISTER_RECHARGE_AMOUNT,
 };
 
 use crate::CANISTER_DATA;
 
-#[update]
+#[update(guard = "is_caller_controller")]
```

Comment:
Not a big fan of guards. It's a 1 liner which could just be
```
if(!is_caller_controller(caller())) return;
```

Makes it easier to unit test this function..?